pkg/services/ngalert/notifier/email_test.go

```diff
@@ -200,7 +200,7 @@ func createSut(t *testing.T, messageTmpl string, subjectTmpl string, emailTmpl *
 		},
 		Message: messageTmpl,
 		Subject: subjectTmpl,
-	}, receivers.Metadata{}, emailTmpl, ns, &images.UnavailableImageStore{}, &alertingLogging.FakeLogger{})
+	}, receivers.Metadata{}, emailTmpl, ns, &images.UnavailableProvider{}, &alertingLogging.FakeLogger{})
```

Comment:
Our interface was previously called `images.ImageStore`. but this name has recently been changed to `images.Provider` in [this PR](https://github.com/grafana/alerting/pull/79) to better reflect its behavior.

All changes from `ImageStore` -> `Provider` respond to this name change in our alerting repository.